### ISSUE TITLE
Ananconda Package Install: conda install -c conda-forge ipyml

### ISSUE THREAD

**Issue by winsumloosesum** [pdeitel/IntroToPython#8]
This is not really an issue but I believe this is an update to the following command:
conda install -c conda-forge ipyml

When I entered the command above I received the following:
PackagesNotFoundError: The following packages are not available from current channels:

  - ipyml

Current channels:

  - https://conda.anaconda.org/conda-forge
  - defaults

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

I went to GitHub:
https://github.com/conda-forge/ipympl-feedstock

I ran each of the 2 commands 1 by 1:
conda config --add channels conda-forge
conda install ipympl

Attached file contains the results.

If these commands are correct please let me know.

Thank you
[Anaconda Update Commands 2 - install ipympl.txt](https://github.com/pdeitel/IntroToPython/files/13766844/Anaconda.Update.Commands.2.-.install.ipympl.txt)

**Comment by pdeitel** [comment]
ipmpl is a typo. It should be ipympl. Where did you see that so I can fix it?

**Comment by winsumloosesum** [comment]
Hi Paul,

I'm not sure what video is was but it is in the Before You Begin lessons
I tried entering the command from the Live Lessons video but received an error.
Maybe this command has been updated??
The command is also in the Intro to Programming - Python
Intro to Programming - Python

Before You Begin

Page
xlvii

Installing jupyter-matplotlib
We implement several animations using a visualization library called Matplotlib. To use
them in Jupyter Notebooks, you must install a tool called ipympl. In the Terminal, Anaconda
Command Prompt or shell you opened previously, execute the following commands1
one at a time:
conda install -c conda-forge ipympl

It could be my error in entering the command correctly.

Thanks
